### DISCUSSION TITLE
Why Telegram?

### DISCUSSION THREAD

**Discussion by zonescape** [XTLS/Xray-core#4059] [Q&A]
In the README one can find [links to official Telegram groups](https://github.com/XTLS/Xray-core#telegram). Not only users but also the developers of this project communicate there. User registration in Telegram is performed by phone number. It means that if you use your real number you can be identified. I don't know about other countries, but in the case of China, we have the disappearance of at least @VictoriaRaymond and @clowwindy. Of course, they could have disappeared of their own free will, but who knows. In the same README you can see a [warning](https://github.com/XTLS/Xray-core#installation) for users about insecure by default control panels, whereas developers themselves are using Telegram which is also insecure by default. In my opinion secure usage of Telegram means registering with a fake phone number and accessing it via anonymous proxy. I highly doubt that all developers follow these precautions.

**Comment by xfrose** [comment]
I guess it's purely due to historical reasons. Many Chinese users are on Telegram, with a smaller portion using real phone numbers.

**Comment by vxtls** [comment]
我认为其实Matrix聊天协议更加好，因为它是E2EE，并且是去中心化的

**Reply by zonescape** [reply]
> 我认为其实Matrix聊天协议更加好，因为它是E2EE，并且是去中心化的
> [EN] I think the Matrix chat protocol is actually better because it is E2EE and decentralized.

I am not an advocate of any messenger. I just think that Telegram is a bad choice. Even GitHub Discussions is better because GitHub registration is performed via email.
